packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -1220,6 +1220,9 @@ export class DataTable {
             class={{
               ["table-cell"]: true,
               [`table-density-${this.density}`]: this.notDefaultDensity(),
+              ["with-overflow"]:
+                columnProps?.dataType === "element" &&
+                isSlotUsed(this.el, cellSlotName),
```

Comment:
Wondering whether this would work on elements that are passed through using html strings (therefore not using the slot)